src/util/symbols.js

```diff
@@ -1,9 +1,13 @@
-export const connected = '____skate_connected';
-export const created = '____skate_created';
-export const name = '____skate_name';
-export const props = '____skate_props';
-export const ref = '____skate_ref';
-export const renderer = '____skate_renderer';
-export const rendering = '____skate_rendering';
-export const rendererDebounced = '____skate_rendererDebounced';
-export const shadowRoot = '____skate_shadowRoot';
+function createSymbol(name) {
```

Comment:
I'd be a fan of randomly generating a string here, even if it's done simply so you don't have to pass it in everywhere.